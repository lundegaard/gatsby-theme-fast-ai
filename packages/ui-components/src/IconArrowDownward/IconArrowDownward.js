import React from 'react';

import SvgIcon from '../SvgIcon';

const IconArrowDownward = ({ ...rest }) => (
	<SvgIcon {...rest}>
		<rect width="24" height="24" opacity="0" />
		<path d="M12 16a1 1 0 0 1-.64-.23l-6-5a1 1 0 1 1 1.28-1.54L12 13.71l5.36-4.32a1 1 0 0 1 1.41.15 1 1 0 0 1-.14 1.46l-6 4.83A1 1 0 0 1 12 16z" />
	</SvgIcon>
);

export default IconArrowDownward;
