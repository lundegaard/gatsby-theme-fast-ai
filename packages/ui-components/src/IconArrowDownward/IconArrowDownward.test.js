import React from 'react';
import { shallow } from 'enzyme';

import IconArrowDownward from './IconArrowDownward';

describe('<IconArrowDownward />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<IconArrowDownward {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
