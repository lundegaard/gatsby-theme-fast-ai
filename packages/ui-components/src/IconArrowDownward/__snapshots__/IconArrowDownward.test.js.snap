// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconArrowDownward /> matches snapshot when shallowly rendered 1`] = `
<SvgIcon>
  <rect
    height="24"
    opacity="0"
    width="24"
  />
  <path
    d="M12 16a1 1 0 0 1-.64-.23l-6-5a1 1 0 1 1 1.28-1.54L12 13.71l5.36-4.32a1 1 0 0 1 1.41.15 1 1 0 0 1-.14 1.46l-6 4.83A1 1 0 0 1 12 16z"
  />
</SvgIcon>
`;
